app/assets/stylesheets/views/mainstream_browse_pages.scss

```diff
@@ -65,6 +72,7 @@ h1 .type {
 }
 
 .attributes {
+  // scss-lint:disable PlaceholderInExtend
   @extend .panel;
   @extend .panel-default;
 
```

Comment:
Whenever you call `scss-lint:disable (rule)` I think that means that particular rule is disabled from that line until either the end of the file or if you add `scss-lint:enable (rule)`. As general good practice we should isolate these disables by wrapping only the specific exception in a disable/enable block.

Reply:
because this is on the block it only disables it on that block nothing else. https://github.com/brigade/scss-lint#disabling-linters-via-source

Reply:
That's a good point - I didn't realise that. However I think there's a few other examples in this file where we should be closing the lint disabler after we've opened it, because it's applying to more than just a small chunk like this.